src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.cs

```diff
@@ -994,89 +996,94 @@ private void UpdateMediaControlState(DependencyProperty property)
 					break;
 
 				case var _ when property == IsRepeatButtonVisibleProperty:
-					BindVisibility(m_tpRepeatButton, IsRepeatButtonVisible);
+					BindVisibility(m_tpRepeatButton, IsImplemented(typeof(MediaPlayer), "IsLoopingEnabled") && IsRepeatButtonVisible, property);
 					break;
 				case var _ when property == IsRepeatEnabledProperty:
 					BindIsEnabled(m_tpRepeatButton, IsRepeatEnabled);
 					break;
 				case var _ when property == IsVolumeButtonVisibleProperty:
-					BindVisibility(m_tpTHVolumeButton, IsVolumeButtonVisible);
+					BindVisibility(m_tpTHVolumeButton, IsImplemented(typeof(MediaPlayer), "Volume") && IsVolumeButtonVisible, property);
 					break;
 				case var _ when property == IsVolumeEnabledProperty:
 					BindIsEnabled(m_tpTHVolumeButton, IsVolumeEnabled);
 					break;
 				case var _ when property == IsFullWindowButtonVisibleProperty:
-					BindVisibility(m_tpFullWindowButton, IsFullWindowButtonVisible);
+					BindVisibility(m_tpFullWindowButton, IsFullWindowButtonVisible, property);
 					break;
 				case var _ when property == IsFullWindowEnabledProperty:
 					BindIsEnabled(m_tpFullWindowButton, IsFullWindowEnabled);
 					break;
 				case var _ when property == IsZoomButtonVisibleProperty:
-					BindVisibility(m_tpZoomButton, IsZoomButtonVisible);
+					BindVisibility(m_tpZoomButton, IsZoomButtonVisible, property);
 					break;
 				case var _ when property == IsZoomEnabledProperty:
 					BindIsEnabled(m_tpZoomButton, IsZoomEnabled);
 					break;
 				case var _ when property == IsPlaybackRateButtonVisibleProperty:
-					BindVisibility(m_tpPlaybackRateButton, IsPlaybackRateButtonVisible);
+					BindVisibility(m_tpPlaybackRateButton, IsImplemented(typeof(MediaPlayer), "PlaybackRate") && IsPlaybackRateButtonVisible, property);
```

Comment:
On platforms where this is not implemented, it will throw `NotImplementedException`, so can we do it like this?

```
private static bool IsImplemented<T>(Func<T> propertyGetter)
{
    try
    {
        _ = propertyGetter();
        return true;
    }
    catch (NotImplementedException)
    {
        return false;
    }
}
```

the call will be `IsImplemented(() => MediaPlayer.PlaybackRate)`
I think this will be more efficient than ApiInformation which uses reflection

Reply:
I thought about that, but my fear was that some attributes that go to the extension (in the case of wasm or gtk) and reach other objects pass through and present errors when called before the element is created.
For example CurrentPosition (goes to the html to validate and displays an error if it doesn't exist yet)
So I tried to make a generic function.
But, it already existed.

Reply:
What I mean is that when trying to get the wasm resource, the get goes to the extension, which goes to the player, which takes the video element from the html.
And when there is still no source it shows an error.
This is not the case for the remaining validations, so we can switch to try/catch. 
But I tried to make something generic and reusable.

Reply:
Ah got it. @jeromelaban What do you think?